src/lib/plot/contour.typ

```diff
@@ -5,11 +5,17 @@
 // Find contours of a 2D array by using marching squares algorithm
 //
 // - data (array): 2D float array (rows => columns)
```

Comment:
This could be a bit more verbose. maybe something like "A 2D array of floats where the first index is the row and the second index is the column" or "An array of arrays of floats where the array of floats is a column and array of arrays is the row". Maybe give an example and describe what the data should represent.